example/presto_cluster/main.tf

```diff
@@ -69,12 +57,12 @@ module "minio" {
   port                            = 9000
   container_image                 = "minio/minio:latest" # todo: avoid using tag latest in future releases
   vault_secret                    = {
-                                      use_vault_provider   = false,
-                                      vault_kv_policy_name = "",
-                                      vault_kv_path        = "",
-                                      vault_kv_access_key  = "",
-                                      vault_kv_secret_key  = ""
-                                    }
+    use_vault_provider   = false,
+    vault_kv_policy_name = "",
+    vault_kv_path        = "",
+    vault_kv_access_key  = "",
+    vault_kv_secret_key  = ""
+  }
```

Comment:
My beautiful spaces :scream:

Reply:
Oh, I thought it was something weird with the formatting 😅 I'm down to change it back if you guys think it is easier to read the other way?

Reply:
If you have any doubts, run `terraform fmt -recursive` <- a community standard

Reply:
Didn't think of that, thanks

Reply:
I was just joking, it doesn't matter really 😆